packages/gatsby-remark-graphviz/src/index.js

```diff
@@ -11,13 +11,19 @@ module.exports = async ({ markdownAST }, pluginOptions = {}) => {
   let codeNodes = []
 
   visit(markdownAST, `code`, node => {
-    const chunks = (node.lang || ``).match(
-      /^(?<lang>\S+)(\s+(?<attrString>.+))?/
-    )
+    const chunks = (node.lang || ``).match(/^(\S+)(\s+(.+))?/)
+
+    if (!chunks || !chunks.length) {
+      return node
+    }
+
```

Comment:
```suggestion
hardcoded to indices 1 and 3 because we can't use named capture groups - those work in node 10+ and we don't transpile it right now
```